assets/src/apps/authoring/components/AdaptivityEditor/ConditionItemEditor.tsx

```diff
@@ -127,7 +127,7 @@ const ConditionItemEditor: React.FC<ConditionItemEditorProps> = (props) => {
             handleFactChange(e.target.value);
             dispatch(setCurrentPartPropertyFocus({ focus: true }));
           }}
-          title={fact.toString()}
+          title={fact?.toString()}
```

Comment:
you sure you want these to be undefined when this is the case? could keep it a string like:
`{fact?.toString() ?? ""`

Reply:
I checked them but it wasn't showing undefined. Anyway, I have made the changes.